fastapi/types.py

```diff
@@ -0,0 +1 @@
+from starlette.types import Receive  # noqa: F401
```

Comment:
Repeated?

Reply:
Yes--thank you! (Though it was actually the one in `fastapi/requests.py` that was repeated)